gobyexample/404.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
linguist-vendored

Reply:
nope, sorry - some are manually edited :(

Reply:
Currently, the vast majority of files appear to be unmodified by human. It might be worth `linguist-vendor`-ing all files and `linguist-vendor=false` whichever ones you've modified.